haml/_table.html.haml

```diff
@@ -0,0 +1,16 @@
+.cads-table-container
+  %table.cads-table{ class: [table_data['caption'] && !table_data['caption'].empty? && 'cads-table__has-caption'] }
+    -if table_data['caption'] and not table_data['caption'].empty?
+      %caption= table_data['caption']
+
+    %tbody
+      %tr
+        - table_data['header'].each do |head|
```

Comment:
Is there a reason this isn't in a `thead`?

Reply:
Thead didn't seem to work that well for voiceover. Reading around it looks like it is more for styling/printning, with a normal TR voiceover read the columns better. I may be wrong though, need someone to test this stuff better.

Reply:
That's fair, makes sense. we can always add it in if future features require it.